Scripts/Source/minai_Sex.psc

```diff
@@ -62,8 +69,8 @@ Function Maintenance(minai_MainQuestController _main)
     Main.Error("Could not find ostim toggle")
   EndIf
   
-  ; Reset incase the player quit during a sex scene or this got stuck
-  SetSexSceneState("off")
+  ; clean any threads from table
+  UpdateThreadTable("clean")
```

Comment:
I rewrote logic on php, so it will look if HERIKA_NAME is in any of current threads - which means they are in sex scene
https://github.com/MinLL/MinAI/pull/49/files#diff-960fdc3c725d063fc8bde5fc727dac8f3b80af66e9be28f9c5b4a65d72ded0a2R91

Reply:
Yeah. AIFF itself has integrations with this particular option inside it's TTS implementation that pre-date my mod. If we set this, the TTS will behave differently.

Reply:
oh I see... sorry didn't notice you are talking about AIFF and not your mod )) Since we have requests to call on start/end of scene I can add it only on php side, there is no need to call it from papyrus.

Reply:
returned SetSexSceneState(via papyrus, in case opt_conf will be changed by ai_ff) and all places it was called